### ISSUE TITLE
Normalization

### ISSUE THREAD

**Issue by ghost** [lizeng614/SqueezeNet-Neural-Style-Pytorch#4]
Can you please explain the reason for using normalization factors of ([0.5, 0.5, 0.5], [0.5, 0.5, 0.5]) ?

Pytorch documents mention that for all pre-trained models,
"The images have to be loaded in to a range of [0, 1] and then normalized using mean=[0.485, 0.456, 0.406] and std=[0.229, 0.224, 0.225]"

**Comment by CoderInPajamas** [comment]
Thank you for pointing it out! I didn't read that docu carefully so I just used a normalization factor I used in tensorflow before. of course you can use the normalization factors in Pytorch documents and and I think there wont be a big difference!
